src/layouts/index.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import { Outlet, NavLink, useNavigate } from "react-router-dom";
+
+const Layout = () => {
+  const navigate = useNavigate();
```

Comment:
If the back button was not working, maybe move the const navigate outside of the function. I remember all the examples of it were outside the nested functions.